rook/scripts/setup_kubespray.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -eu
+
+# change into kubespray dir
+cd kubespray/
+
+# install the requirements
+pip install -r requirements.txt
```

Comment:
we mentioned this offline so adding it here for the record. Instead of having to install kubespray dependencies when the workflow is executed, we can have an image for it. This would also allow us to not have to add kubespray as a submodule. This would be similar to how it's done for the [ceph-ansible](https://github.com/getpopper/library/tree/master/ceph-ansible) image.